Nikita_Korzhavin/api/products.js

```diff
@@ -0,0 +1,16 @@
+// product используются для моделирования начинок, типов салатов и напитков
+function Product(price, calories, name) {
+  this.price = price || 0;
+  this.calories = calories || 0;
+  this.name = name;
```

Comment:
Why not set default name (at least `''`, still better than `undefined`) if you have default values for other arguments?

Reply:
fixed